OOP part 1/code/src/Bow.ts

```diff
@@ -0,0 +1,15 @@
+import { Weapon } from "./Weapon";
+let bowName = "bow";
+export class Bow extends Weapon
+{
+    constructor(baseDamage,baseDurability,value,weight)
+    {
+        super(bowName,baseDamage,baseDurability,value,weight);
+    }
+    polish() : void
```

Comment:
Always use "{" opening brace after function name not in new line
e.g.   polish() : void {
              ///
          }